### PR TITLE
chore: hlx3 config changes

### PR DESCRIPTION
**Pull request by davidnuescheler**
https://hlx3--adobe-io-website--adobe.hlx3.page/apis

vs. 

https://main--adobe-io-website--adobe.hlx.page/apis

### CODE REVIEW

**Comment by davidnuescheler** [comment]
https://hlx3--adobe-io-website--adobe.hlx3.page/lightroom/
https://hlx3--adobe-io-website--adobe.hlx3.page/animate/
https://hlx3--adobe-io-website--adobe.hlx3.page/audition/
https://hlx3--adobe-io-website--adobe.hlx3.page/mobile-sdk/
https://hlx3--adobe-io-website--adobe.hlx3.page/helix-demo/
https://hlx3--adobe-io-website--adobe.hlx3.page/developer-console/
https://hlx3--adobe-io-website--adobe.hlx3.page/UMAPI/
https://hlx3--adobe-io-website--adobe.hlx3.page/audience-manager/
https://hlx3--adobe-io-website--adobe.hlx3.page/photoshop/api/
https://hlx3--adobe-io-website--adobe.hlx3.page/project-firefly-helix/
https://hlx3--adobe-io-website--adobe.hlx3.page/adobesign-api/
https://hlx3--adobe-io-website--adobe.hlx3.page/app-builder/
https://hlx3--adobe-io-website--adobe.hlx3.page/open/
https://hlx3--adobe-io-website--adobe.hlx3.page/devecosystem/
https://hlx3--adobe-io-website--adobe.hlx3.page/creative-cloud-events/
https://hlx3--adobe-io-website--adobe.hlx3.page/dc/

File: tools/sidekick/config.js [hunk 1]
```diff
@@ -15,4 +15,5 @@ window.hlx.initSidekick({
   project: 'Adobe I/O',
   host: 'www.adobe.io',
```
Comment (davidbenge) [inline tools/sidekick/config.js:16]: should this still be www.adobe.io?  We moved the name to developer.adobe.com.  In theory.  Both names are bound to same service endpoint in Fastly so there is no real difference there.  Will it make any difference on your side?

**Comment by davidbenge** [comment]
Did you need me to update fastly origin as well?  If so should I update before or after the PR merge?

**Review by davidbenge — APPROVED** [review]
looks good

**Pull request merged**